tensorflow/python/keras/layers/convolutional.py

```diff
@@ -151,7 +167,12 @@ def __init__(self, rank,
   def build(self, input_shape):
     input_shape = tensor_shape.TensorShape(input_shape)
     input_channel = self._get_input_channel(input_shape)
-    kernel_shape = self.kernel_size + (input_channel, self.filters)
+    if input_channel % self.groups != 0:
+      raise ValueError(
+          'The number of input channels is not divisible by the number of '
```

Comment:
"The number of input channels must be evenly divisible by the number of groups. Received groups={}, but the input has {} channels (full input shape is {})"